pkg/graph/schema.graphql

```diff
@@ -606,6 +617,26 @@ updateDiscussionReply(id: UUID!, changes: DiscussionReplyChanges!): DiscussionRe
 
 deleteDiscussionReply(id: UUID!): DiscussionReply!
 @hasRole(role: MINT_BASE_USER)
+
+lockTaskListSection(modelPlanID: UUID!, section: String!): Boolean!
```

Comment:
I think we should have a task list section field here. In your example, we could lock any string we wanted. 

I don't think there is any point to allow users to send a request for a section that doesn't exist.

Reply:
Yep, agreed. Done.